l10n_ch_hr_payroll/models/__init__.py

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
-# © 2012 David Coninckx (Opennet Sarl)
+# © 2016 David Coninckx (Opennet Sarl), Ermin Trevisan (twanda AG)

```

Comment:
Could be better this way:
copyright 2012 David Coninckx (Opennet Sarl)
copyright 2016 Ermin Trevisan (twanda AG)